examples/platformer/main.go

```diff
@@ -127,7 +128,7 @@ func (g *Game) Update() error {
 		g.gopher = &char{x: 50 * unit, y: groundY * unit}
 	}
 
-	// Controls
+	// Controls.
```

Comment:
This is a word, not a sentence. Remove `.`

Reply:
Isn't it supposed to be like that? I thought the changes I made in the other branch were not gone. I don't know if the changes in the previous branch were visible, so I created a new one.